pkg/helmt/helmt.go

```diff
@@ -124,6 +125,11 @@ func template(name string, chart string, values []string, namespace string, skip
 	} else {
 		args = append(args, "--output-dir", ".")
 	}
+	if len(ApiVersions) > 0 {
```

Comment:
not sure if this is needed

Reply:
In case nothing is specified for apiVersions in helm-chart.yaml, the block gets executed and leads to errors